### PR TITLE
creation of  search page natively

### LINKED ISSUE #14: Show search results in this web app instead of routing to musicbrainz.org
We have the Search API available at https://musicbrainz.org/doc/MusicBrainz_API

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

This PR fixes #14 

Right now I have created a separate  header link to go to the search page we can remove that later this is for the demo only.

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
+import React, { useCallback, useState, useEffect } from "react";
+import Header from "../components/Shared/Header";
+import Footer from "../components/Shared/Footer";
+import ScrollToTop from "../components/Shared/ScrollToTop";
+
+import { themeSetting } from "../components/Shared/UISettings";
+import dynamic from 'next/dynamic';
+
+const BarcodeScanner = dynamic(
```
Comment (akshaaatt) [inline pages/search.js:9]: This is only useful for the intro page and we do not need it here.

**Review by akshaaatt — CHANGES_REQUESTED** [review]
We can look to cleanup the page and start working on components for the page.

File: pages/search.js (hunk 1 above)
Comment (asubedy) [reply pages/search.js:9]: Yes alright understood, I was trying to copy the intro page functionalities so thought it was necessary .

**Comment by asubedy** [comment]
Oops I think the code for fetching the data from the api is also removed. @akshaaatt

**Comment by akshaaatt** [comment]
While, the work here isn't complete, I am merging this because the process is fine so far.

**Pull request merged**